webots_ros2_epuck/launch/rats_life_waypoints_launch.py

```diff
@@ -0,0 +1,170 @@
+
+    collection.add(position=[0, 0], orientation=0)      # Initial pose
+    collection.add(orientation=-pi/2)                   # Explore the hall, rotate
+    collection.add(orientation=0)                       # Explore the hall, rotate
+    collection.add(position=[0.28, 0], orientation=0)   # Move near RED entrance
+    collection.add(orientation=pi/2)                    # Rotate towards RED
+    collection.add(position=[0.28, 0.2])                # Move inside RED
+    collection.add(orientation=0)                       # Explore RED, rotate
+    collection.add(orientation=-pi)                     # Explore RED, rotate
+    collection.add(orientation=-pi/2)                   # Rotate towards BLUE
+    collection.add(position=[0.23, -0.2])               # Move inside BLUE
+    collection.add(orientation=0)                       # Explore BLUE, rotate
+    collection.add(orientation=pi/3)                    # Explore BLUE, rotate
+    collection.add(orientation=-pi/2)                   # Explore BLUE, rotate
+    collection.add(orientation=-2.7*pi)                 # Explore BLUE, rotate
+    collection.add(orientation=pi/2)                    # Rotate towards the hall
+    collection.add(position=[0.23, 0])                  # Go to the hall
+    collection.add(orientation=1/3*pi)                  # Explore the hall, rotate
+    collection.add(orientation=pi)                      # Rotate towards back
+    collection.add(position=[-0.1, 0])                  # Go to the GREEN entrance
+    collection.add(orientation=pi/2)                    # Explore hall, rotate
+    collection.add(orientation=pi/4)                    # Explore hall, rotate
+    collection.add(orientation=2/3*pi)                  # Explore hall, rotate
+    collection.add(orientation=-pi/2)                   # Rotate towards the GREEN
+    collection.add(position=[-0.1, -0.2])               # Move inside GREEN
+    collection.add(orientation=0)                       # Explore GREEN, rotate
+    collection.add(orientation=pi/3)                    # Explore GREEN, rotate
+    collection.add(orientation=-pi/2)                   # Explore GREEN, rotate
+    collection.add(orientation=3/4*pi)                  # Explore GREEN, rotate
+
+    return collection.export()
+
+
+def generate_launch_description():
+    use_sim_time = LaunchConfiguration('use_sim_time')
+
+    package_dir = get_package_share_directory('webots_ros2_epuck')
+
+    webots = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            os.path.join(get_package_share_directory('webots_ros2_core'), 'launch', 'robot_launch.py')
+        ),
+        launch_arguments=[
+            ('package', 'webots_ros2_epuck'),
+            ('executable', 'driver'),
+            ('world', os.path.join(package_dir, 'worlds', 'rats_life_benchmark.wbt')),
+        ],
+        condition=launch.conditions.IfCondition(use_sim_time)
+    )
+
+    # Launch Navigation2 without a localization and without costmaps
```

Comment:
It would be nice to say why we need it if localization and costmaps is disabled.